pdns/opensslsigners.cc

```diff
@@ -238,10 +237,9 @@ class OpenSSLRSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
 private:
   static int hashSizeToKind(size_t hashSize);
 
-  std::unique_ptr<RSA, void(*)(RSA*)> d_key;
+  std::unique_ptr<RSA, void (*)(RSA*)> d_key;
```

Comment:
RAS_new, RSA_free?

Reply:
Do you mean to use `decltype(&RSA_free)` instead of the raw function pointer type? If yes, this is done in an upcoming PR.

Reply:
yeah